misc/alert_rules.json

```diff
@@ -527,5 +527,10 @@
     "rule": "applications.app_type = \"suricata\" && application_metrics.metric = \".total_error_delta\" && application_metrics.value >= \"2\"",
     "name": "Suricata Packet Error >= 2%",
     "severity": "critical"
+  },
+  {
+    "rule": "applications.app_type = \"mysql\" && applications.app_state != \"OK\"",
```

Comment:
What about other applications?  Also, is "ERROR" guaranteed to be MySQL server not responding?  Or could it be another issue?

Reply:
This could in theory be extended to other application, but my use-case just includes mysql so I made it for that. I can also revert the changes here since it maybe really isn't wort it.

Also ERROR: should be pretended to any error, since that's coming from the php snmpd-extend script. Tested it with server-offline, wrong port, wrong IP and wrong user-credentials. I don't expect there to come anything else.